package.json

```diff
@@ -33,6 +33,7 @@
     "jose": "^5.9.2",
     "lucide-react": "^0.441.0",
     "next": "15.0.0-canary.152",
+    "next-themes": "^0.3.0",
```

Comment:
Let's just use the system setting + tailwind.

Reply:
Good shout. I didn't realise tailwind could do that by itself!